app/src/main/java/edu/gatech/cs2340/hkskh/AccountManager.java

```diff
@@ -0,0 +1,76 @@
+package edu.gatech.cs2340.hkskh;
+
+import java.util.HashMap;
+
+/**
+ * Created by Kirby on 2/16/2018.
+ */
+
+public class AccountManager {
+    private static HashMap<String, User> login = new HashMap<>();
+
+    public AccountManager() {
+    }
+
+    //Will return false if there is no user key, hashmap is empty, or the passed in password is incorrect
+    //Will return true if the user-pass mapping matches what is passed in
```

Comment:
For future reference I'd throw this comment in the doc block.

Reply:
What is the doc block? Do you mean do a javadoc for it?